docs/content/en/docs/troubleshooting/_index.md

```diff
@@ -32,34 +32,81 @@ environment for debugging. The tool is meant to be used for debugging a single
 Tetragon agent node. Note that in the context of Kubernetes, the command needs
 to be run from inside the Tetragon Pod's container.
 
+**Key Information Collected by the Bugtool:**
+
+- Tetragon configuration
+- Network configuration
+- Kernel configuration
+- eBPF maps
+- Process traces (if tracing is enabled)
+
+## Running tetra bugtool Demonstrations
+
+### Kubernetes
+
+1. **Identify Tetragon Pod:**
+
+```bash
+kubectl get pods -n <tetragron-namespace> -l app.kubernetes.io/name=tetragon
+```
+
+2. **Execute tetra bugtool within the Pod:**
+
+```bash
+kubectl exec -n <tetragron-namespace> <tetragron-pod-name> -- tetra bugtool
+```
+
+### Container Installation
+
+1. **Enter the Tetragon Container:**
+
+```bash
+docker exec -it <tetragron-container-id> /bin/sh
+```
+
+2. **Run tetra bugtool:**
+
+```bash
+tetra bugtool
+```
+
+### Systemd Host Installation
+
+1. **Execute tetra bugtool with Elevated Permissions:**
+
+```bash
+sudo tetra bugtool
+```
+
+> Note: Replace <...> placeholders with the actual values applicable to your setup.
+
 ## Enable debug log level
 
 When debugging, it might be useful to change the log level. The default log
 level is controlled by the log-level option at startup:
 
-* Enable debug level with `--log-level=debug`
-* Enable trace level with `--log-level=trace`
+- Enable debug level with `--log-level=debug`
+- Enable trace level with `--log-level=trace`
```

Comment:
please avoid making unnecessary changes especially for git history, this doesn't modify anything when rendered.